Yosemite/Yosemite/Actions/AppSettingsAction.swift

```diff
@@ -68,6 +68,25 @@ public enum AppSettingsAction: Action {
     ///
     case resetFeatureSwitches
 
+    // MARK: - Products Settings
+
+    /// Loads the products settings
+    ///
+    case loadProductsSettings(siteID: Int64, onCompletion: (Result<StoredProductSettings, Error>) -> Void)
+
+    /// Add or Update products settings
+    ///
+    case upsertProductsSettings(siteID: Int64,
+                                sort: String? = nil,
```

Comment:
I think it's fair to move `ProductsSortOrder` to Yosemite since Yosemite is business logic anyway. In this way, we don't need to use `String`. But up to you. I don't know if we have any guidelines on this.